source/help/messaging/attaching-files.rst

```diff
@@ -1,60 +1,56 @@
 
-How to Attach Files
--------------------
-You can attach files in the following ways:
+Attach Files to Messages
+------------------------
 
--  Use the attachment icon - click the paperclip icon inside the message input box
--  Drag and Drop
+You can attach files to messages in the following ways:
+
+-  Use the attachment icon - select the paperclip icon inside the message input box
+-  Drag and drop
 -  Paste from the clipboard
 
-File Previewer
---------------
+Preview File Attachments
+------------------------
+
 Mattermost has a built-in file previewer that you can use to:
 
 -  Download files
 -  Share public links
 -  View media
 
-Click the thumbnail of an attached file to open it in the file previewer.
+Select the thumbnail of an attached file to open it in the file previewer.
 
 Downloading Files
 ~~~~~~~~~~~~~~~~~
-You can download an attached file by clicking the download icon next to the file 
-thumbnail.
+
+You can download an attached file by selecting the download icon next to the file thumbnail.
 
 Sharing Public Links
 ~~~~~~~~~~~~~~~~~~~~
-Public URLs allow you to share attachments with anyone outside the Mattermost 
-system. To share an attachment, click the thumbnail of an attachment, then click
-**Get Public Link**.
-
-.. Tip::
-  If **Get Public Link** is not visible in the file previewer
-  and you prefer that the feature is enabled, you can request your System
-  Admin to enable the feature from the System Console under
-  **Security > Public Links**.
-
-Viewing Media:
-~~~~~~~~~~~~~~~~~~~~~
+
+Public URLs allow you to share attachments with anyone outside the Mattermost system. To share an attachment, select the thumbnail of an attachment, then select **Get Public Link**.
+
+.. tip::
+  If **Get Public Link** is not visible in the file previewer, and you prefer that the feature is enabled, you can request your System Admin to enable the feature from the System Console under **Site Configuration > Public Links**.
+
+Viewing Media
+~~~~~~~~~~~~~~
+
 The following media formats are supported on most browsers:
 
 -  Images: BMP, GIF, JPG, JPEG, PNG, SVG
 -  Video: MP4
 -  Audio: MP3, M4A
 -  Files: PDF, TXT
 
-Other document previews (Word, Excel, PPT) are not yet supported.
+Other document previews (such as Word, Excel, or PPT) are not yet supported.
 
 Attachment Limits and Sizes
 ---------------------------
-Please note that Mattermost supports a maximum of five (5) attached 
-files per post.
 
-The default maximum file size is 50 MB, but this can be changed by the 
-System Admin.
+Mattermost Cloud supports a maximum of 10 attached files per post. Mattermost Team Edition and Enterprise Edition support a maximum of five attached files per post, and this maximum will be increased to 10 files per post in a future release.
```

Comment:
@cwarnermm @justinegeffen how are we handling the docs when it comes to cloud vs on-prem in general? I would find this statement strange as an on-prem admin. Perhaps docs should be versioned to allow admins to switch to the docs appropriate for their server version or cloud (default)

> this maximum will be increased to 10 files per post in a future release

Reply:
@esethna @justinegeffen @wiersgallak - The ability to version MM product documentation is a high priority project that will time to implement. This is the proposed interim approach that is intended to be temporary. This statement is scheduled to be removed as part of v5.31 documentation updates.

Reply:
@esethna The documentation now clarifies when the feature will be available for self-hosted deployments.